tests/models/hf_dict.py

```diff
@@ -1,17 +0,0 @@
-import torch
-
-
-class ModelData:
-    def __init__(self, input_shape, torch_dtype: torch.dtype, xfail: bool = True):
-        self.xfail = xfail
-        self.input_shape = input_shape
-        self.dtype = torch_dtype
-
-
-model_dict = {
```

Comment:
Why remove this? Might be good to keep these tests so we can start working to generalize to all HF models

Reply:
I plan to bring it back once any of it works, but these crash pytest currently even as xfails

Reply:
I see, makes sense